### PR TITLE
Simplifying build, using build tasks

### PR DESCRIPTION
**Pull request by MadsAndreasen**
Using the tasks to build from the meson.build command. Making it in line with the other commands.

### CODE REVIEW

File: src/meson/runners.ts [hunk 1]
```diff
@@ -76,45 +76,17 @@ export async function runMesonReconfigure() {
...
-      if (code !== 0) {
-        throw new Error(
-          "Build failed. See Meson Build output for more details."
-        );
-      }
+  try {
+    await vscode.tasks.executeTask(await getTask("build", name));
+  } catch (e) {
+    vscode.window.showErrorMessage(`Couldn't build ${name}`);
+    getOutputChannel().appendLine(`Building targer ${name}:`);
```
Comment (jfsimoneau) [inline src/meson/runners.ts:84]:
```suggestion
    getOutputChannel().appendLine(`Building target ${name}:`);
```

**Review by Ongy — APPROVED** [review]
Less duplication makes for a happy developer :)

**Comment by MadsAndreasen** [comment]
> Less duplication makes for a happy developer :)

And together with the other PR I have, will provide compiler errors in the problem view.

**Comment by Ongy** [comment]
@dcbaker can we get this merged? I think for this we also don't need a changelog entry as it's not user facing at first.

File: src/meson/runners.ts [hunk 2]
```diff
@@ -76,45 +76,17 @@ export async function runMesonReconfigure() {
...
-      const code = await stream.finishP();
-      if (code !== 0) {
-        throw new Error(
-          "Build failed. See Meson Build output for more details."
-        );
-      }
+  try {
+    await vscode.tasks.executeTask(await getTask("build", name));
+  } catch (e) {
+    vscode.window.showErrorMessage(`Couldn't build ${name}`);
```
[the remark this answers is not in the document]
Comment (MadsAndreasen) [reply src/meson/runners.ts]: I'ts a reuse of the original text.

But I agree - I've also changed the wording for configuration.

**Pull request merged**